Docs/debugging.md

```diff
@@ -0,0 +1,77 @@
+# Debugging
+
+The content of this guide will help you to configure PHPStorm in order to debug a PHP Application.
+The following documentation is tested with Linux, but might be easily adaptable for other host systems.
+
+## Container config
+
+*Xdebug* is deactivated by default. If you want to activate it, you need to set the following environment variables.
+
+In this case we use a part of the default configuration of a php-fpm and httpd application but this is the same vein for others versions:
+```yaml
+services:
+  fpm:
+    image: akeneo/fpm:php-7.1
+    environment:
+      PHP_IDE_CONFIG: 'serverName=cli' # Your will need the server name for your IDE config
+      PHP_XDEBUG_ENABLED: 1 # Set to 0 if you don't want to activate xdebug
+      PHP_XDEBUG_IDE_KEY: 'XDEBUG_IDE_KEY' # the IDE KEY you want to use
+      XDEBUG_CONFIG: 'remote_host=xxx.xxx.xxx.xxx' # Your host IP address, this one is needed only if the docker daemon run on TCP instead of unix domain socket (on Mac OS for instance)
```

Comment:
Didn't saw this. Nope, this is mandatory for CLI debugging. I removed the Mac OS specificities from those example.